### ISSUE TITLE
results-date

### ISSUE THREAD

**Issue by MMAKENPO** [fayyaadh4/itunes_search#1]
hello from Greece!!!happy and blessed new year for you and your family...my friend many congratulationsssssssssssss for this amazing web.greattttttttttt job!my friend I am searching for many months one web like yours!!!but please could you make it increase unlimited results?it would be great!!!I know itunes have limit but you as developer I am sure you can create one button load more !!!and pleaseeeeeeeee I am begging you can you make it appear also date for every song or year?!!!God bless you!!! unfortunately I have no idea from developing and programming!!! pleaseeeeeeeeeeeee............ I love spanish music so muchhhhhhhhhhhh especially pop,balada!!!please I am sure there is some way to increase unlimited results!!!one button load more for example....and date I mean one feature search by year or date...its up to you!!!I AM BEGGING YOUUUUUUUUUU....

**Comment by fayyaadh4** [comment]
Hi thanks for the feedback. I'll make the changes next month as I dont have
much time this month. I'll send you an email when I've updated it and could
you please remind me the end of the month?

Regards,
Fayyaadh

**Comment by MMAKENPO** [comment]
heyyyyyyyyyyyy thanks so much where answer me!!!!oooooooooo you late so much?what a pity....my friend is so difficultult make a load more button today?I am begging youuuuuuuuuuuuuuuuuuuuuu man!!!I love spanish music so so much and  I would fine all the titles and songs!!!

**Comment by fayyaadh4** [comment]
Okay fine. I added a load more button and added the release date to each
item. Hope you enjoy it more and are able to make more use of it

**Comment by MMAKENPO** [comment]
woowwwwwwwwwwwwwwwwwwwwwwwwwwwww.....................my friendddddddddddddd pleaseeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee unlimitedddddddddddddddddddddddddd...I AM BEGGING YOUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUU

**Comment by fayyaadh4** [comment]
It cant. The API which stores the data only allows 200 max

**Comment by MMAKENPO** [comment]
OOOOOOOOOOOO MY FRIEND I KNOWWWWWWWWW THIS!!!BUT I AM SURE YOU CAN INCREASE UNLIMITED!!!!PLEASE DUDE!!!

**Comment by MMAKENPO** [comment]
I LOVE YOUR WEBBBBBBBBBBB....AMAZINGGGGGGGGGGGG DESIGN AND BIG IMAGES!!!!THE ONLY WHERE MISS IS THE UNLIMITED RESULTS!!!

**Comment by fayyaadh4** [comment]
As you can see the range for the limit is from 1 to 200 unfortunately

**Comment by MMAKENPO** [comment]
YES,I KNOW MY FRIEND THAT ITUNES HAVE LIMIT...BUT YOU AS DEVELOPER YOU CAN INCREASE UNLIMITED!!!ITS POSSIBLE THIS MY FRIEND!!!PLEASE FORGIVE MY PATIENCE BUT I LOVE SPANISH MUSIC SO MUCH ADN IT WOULD BE GREAT FIND ALL THE SONGS...THATS WHY!!!

**Comment by MMAKENPO** [comment]
PLEASE MY GOOD FRIEND!!!

**Comment by MMAKENPO** [comment]
BUT MY GOOD FRIEND DO YOU CHECK THE LINK WHERE I SEND YOU?THIS APPEAR UNLIMITED....BUT ITS NOT LIKE YOURSSSSSSSSS!!!!YOURS IS AMAZINGGGGGGGGGGGGGGGGGGGGG

**Comment by MMAKENPO** [comment]
PLEASE TRY IT.....I AM SURE YOU WILL MAKE IT!!!!!!!!!!!!

**Comment by MMAKENPO** [comment]
OK MY FRIEND!!!I DONT WANT THINK THAT I AM PRESSURE!!!I TRUST YOUR KNOWLEDGE AND I AM SURE YOU CAN MAKE IT!!!!BUT ITS UP TO YOU....TAKE CARE AND GOD BLESS YOU!!!

**Comment by MMAKENPO** [comment]
GOOD MORNING FROM GREECE!!!MY FRIEND DO YOU CHECK THE WEB WHERE I SEND YOU WITH UNLIMITED RESULTS?IF THE PERSON WHERE BELONGS THIS PROJECT CAN SHOWS UNLIMITED RESULTS,THEN I AM SURE YOU CAN TOO ALSO!!!I LOVE YOUR WEB SO MUCHHHHHHHHHHHHHH....UNFORTUNATELY I AM NOT RICH TO PAY YOU TO MAKE IT APPEAR UNLIMITED!!!I AM BEGGING YOU MAN...I LIKE SO MUCH SPANISH MUSIC!!!

**Comment by MMAKENPO** [comment]
please my good friend!!!can you try change the limit to unlimited?I am begging you!!!
